src/components/metadata.svelte

```diff
@@ -1,20 +1,33 @@
 <script lang="ts">
-  import { MetaTags, type MetaTagsProps } from 'svelte-meta-tags';
-  import type { Seo } from '../data/types';
+  import { MetaTags } from 'svelte-meta-tags';
   import dedent from 'dedent';
   import { urlFor } from '../data/sanity-client-browser';
   import type { Picture } from 'vite-imagetools';
+  import type { Metadata } from '../data/types';
 
-  export let title: string | undefined = undefined;
-  export let description: string | undefined = undefined;
-  export let data: MetaTagsProps = {};
-  export let seo: Seo | null = null;
+  export let data: Metadata = {
+    title: null,
```

Comment:
This field is currently not defined in the schema in sanity, do we want to add it there, perhaps?